mkdocs.yml

```diff
@@ -52,14 +52,15 @@ plugins:
 
 nav:
   - Overview: index.md
-  - Getting Started: common/getting_started.md
+  - Getting started: common/getting_started.md
   - Architecture: common/architecture.md
   - Configuration: common/configuration.md
   - Customization: common/customization.md
-  - Location Services: common/location_services.md
-  - Achieving Low Power: common/low_power.md
-  - nRF Cloud Fota: common/fota.md
-  - Testing and CI Setup: common/test_and_ci_setup.md
+  - Provisioning: common/provisioning.md
+  - Location services: common/location_services.md
+  - Strategies for power consumption: common/low_power.md
```

Comment:
IMO **Achieving Low Power** is a better name and more true to what the doc section is trying to achieve.